web/vite.config.cts

```diff
@@ -41,6 +41,50 @@ const sentryPluginOptions: SentryVitePluginOptions = {
   },
 };
 
+const PWAManifest: Partial<ManifestOptions> = {
+  name: 'Electricity Maps',
+  short_name: 'Electricity Maps',
+  start_url: '/',
+  display: 'standalone', // For some reason, the type is not recognized without the cast
```

Comment:
Is this comment still valid? I don't get the "cast" thing at least :)

Reply:
Ah no, I fixed that in another way 😅

I'll remove it.